ed25519/src/lib.rs

```diff
@@ -215,21 +173,19 @@ mod test {
 		let public = pair.public();
 		let message = b"Something important";
 		let signature = pair.sign(&message[..]);
-		assert!(signature.verify(&message[..], &public));
+		assert!(verify_strong(&signature, &message[..], &public));
```

Comment:
why `_strong` prefix?

Reply:
the function `verify` already existed, this one is just with stronger types